Sofa/Component/LinearSolver/Direct/src/sofa/component/linearsolver/direct/BTDLinearSolver.h

```diff
@@ -45,6 +45,9 @@ class BTDLinearSolver : public sofa::component::linearsolver::MatrixLinearSolver
 public:
     SOFA_CLASS(SOFA_TEMPLATE2(BTDLinearSolver, Matrix, Vector), SOFA_TEMPLATE2(sofa::component::linearsolver::MatrixLinearSolver, Matrix, Vector));
 
+    using Inherit = sofa::component::linearsolver::MatrixLinearSolver<Matrix, Vector>;
+    using Inherit::initData;
```

Comment:
```suggestion
    using Inherit1::initData;
```

Reply:
Yeah I know... I tried investigating that but I really did not find anything.
The only hint I got is the fact that all the stuff inheriting MatrixLinearSolver seems to need it.